messages/en.js

```diff
@@ -100,13 +100,13 @@ Blockly.Msg.OBJECT_KEYS_TOOLTIP = "Get all keys from the object";
 // Extra blocks
 // =========================================================================================
 Blockly.Msg.TEXT_SPECIAL_CHARACTER = "special character %1";
-Blockly.Msg.TEXT_SPECIAL_CHARACTER_TOOLTIP = "A special character that shouldn't be escaped (by a '\' prefix";
+Blockly.Msg.TEXT_SPECIAL_CHARACTER_TOOLTIP = "A special character that shouldn’t be escaped (by a '\\' prefix)";
 Blockly.Msg.TEXT_SPECIAL_CHARACTER_LF = "line feed (\\n)";
```

Comment:
In the Japanese translation, there was a good idea to add in parentheses (LF), (CR), (CRLF), before the (\n), (\r), (\r\n), since it is the ASCII symbol name. However, it adds more information, whereas Blockly is intended to be really simple. It's an open discussion here

Reply:
It makes sense to me to add that, since it are ASCII symbol names like you said.
Unless @cymplecy has an objection for some reason?

Reply:
Fine with me

Reply:
Did it.